roles/ocp_netobserv/tasks/setup.yml

```diff
@@ -0,0 +1,215 @@
+              requests:
+                cpu: "{{ netobserv_agent_cpu | default(ocp_netobserv_agent_cpu) }}"
+                memory: "{{ netobserv_agent_memory | default(ocp_netobserv_agent_memory) }}"
+            sampling: {{ netobserv_sampling | default(ocp_netobserv_sampling) }}
+          type: EBPF
+        consolePlugin:
+          autoscaler:
+            maxReplicas: {{ netobserv_console_max_replicas | default(ocp_netobserv_console_max_replicas) }}
+            metrics:
+              - resource:
+                  name: cpu
+                  target:
+                    averageUtilization: {{ netobserv_console_avg_utilization | default(ocp_netobserv_console_avg_utilization) }}
+                    type: Utilization
+                type: Resource
+            minReplicas: 1
+          enable: true
+          imagePullPolicy: IfNotPresent
+          logLevel: info
+          port: 9001
+          portNaming:
+            enable: true
+            portNames:
+              "3100": loki
+          quickFilters:
+            - default: true
+              filter:
+                dst_namespace!: openshift-,netobserv
+                src_namespace!: openshift-,netobserv
+              name: Applications
+            - filter:
+                dst_namespace: openshift-,netobserv
+                src_namespace: openshift-,netobserv
+              name: Infrastructure
+            - default: true
+              filter:
+                dst_kind: Pod
+                src_kind: Pod
+              name: Pods network
+            - filter:
+                dst_kind: Service
+              name: Services network
+          register: true
+          replicas: 1
+          resources:
+            limits:
+              memory: "{{ netobserv_console_limits_memory | default(ocp_netobserv_console_limits_memory) }}"
+            requests:
+              cpu: "{{ netobserv_console_cpu | default(ocp_netobserv_console_cpu) }}"
+              memory: "{{ netobserv_console_memory | default(ocp_netobserv_console_memory) }}"
+        loki:
+          authToken: FORWARD
+          statusUrl: "https://netobserv-loki-query-frontend-http.netobserv.svc:3100/"
+          tenantID: network
+          timeout: 10s
+          tls:
+            caCert:
+              certFile: service-ca.crt
+              name: netobserv-loki-gateway-ca-bundle
+              namespace: "netobserv"
+              type: configmap
+            enable: true
+            insecureSkipVerify: {{ netobserv_loki_tls_insecure_skip_verify | default(ocp_netobserv_loki_tls_insecure_skip_verify) }}
+            userCert:
+              namespace: "netobserv"
+          url: "https://netobserv-loki-gateway-http.netobserv.svc:8080/api/logs/v1/{{ netobserv_bucket | default(ocp_netobserv_bucket) }}"
+        processor:
+          conversationHeartbeatInterval: 30s
+          debug: {}
+          healthPort: 8080
```

Comment:
should we define ports as parameters with default values to avoid potential port conflicts?

Reply:
Those are internal ports, I do not think it is necessary to allow such level of configuration. Thanks!